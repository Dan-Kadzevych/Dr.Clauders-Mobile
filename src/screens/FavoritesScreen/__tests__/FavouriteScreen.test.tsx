import React from 'react';
import renderer from 'react-test-renderer';

import F from '../FavoritesScreen';

describe('Favourite Screen', () => {
  test('Renders correctly', () => {
    const component = renderer.create(<F />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
