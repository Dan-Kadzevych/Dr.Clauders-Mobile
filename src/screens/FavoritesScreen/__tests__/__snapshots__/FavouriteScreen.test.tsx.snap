// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favourite Screen Renders correctly 1`] = `
<View
  onLayout={[Function]}
  pointerEvents="box-none"
  style={
    Object {
      "paddingBottom": 0,
      "paddingLeft": 0,
      "paddingRight": 0,
      "paddingTop": 20,
    }
  }
>
  <Text>
    FavoriteScreen
  </Text>
</View>
`;
